test/junos/test_confirm_commit.py

```diff
@@ -0,0 +1,119 @@
+"""Test confirm commit calls and results"""
```

Comment:
This is **very** interesting. Would you mind looking into generalizing this set of tests? Configuration related parameters are here: https://github.com/napalm-automation/napalm/blob/develop/napalm/base/test/base.py

The goal is to make sure all drivers test the same workflow to ensure they behave consistently. So it would be great if the tests were defined in a common file and each driver would just have to define the mocked data.

Reply:
Sure, let me look and see what I can do.

Reply:
@sdanelli No worries...I just didn't want you doing work that we might not keep.

Reply:
@ktbyers thanks, FWIW it explains the approach I took for Junos on this PR, but I agree, if you are testing against real devices, this seems like a waste.

Reply:
Yeah, let's hold onto this.